README.md

```diff
@@ -1,6 +1,5 @@
 # Babitz
-
-This application is generated using [LoopBack 4 CLI](https://loopback.io/doc/en/lb4/Command-line-interface.html) with the
+This application is the api server for Babitz. This application is created using [LoopBack 4 CLI](https://loopback.io/doc/en/lb4/Command-line-interface.html) with the
```

Comment:
Please add a link to the main repo, see the README for the frontend repo for reference.

Reply:
This is still pending, please check out https://github.com/pesto-students/babitz-frontend/blob/main/README.md for reference.